src/styles/index.css

```diff
@@ -19,3 +19,7 @@ svg.recharts-surface {
 .recharts-cartesian-grid-horizontal line:last-child {
   stroke-opacity: 0;
 }
+
+html {
```

Comment:
(i'd prefer if this appeared higher in the file)

Reply:
OK, moved to the top (after the imports)